assets/css/styleguide.scss

```diff
@@ -43,9 +43,31 @@ a#skipnav {
   }
 }
 
+/* Fixes anchor position when header is fixed */
+:target:before {
+  content: "";
+  display: block;
+  height: 75px;
+  margin-top: -75px;
+
+  @include media($medium-screen) {
+    height: 90px;
+    margin-top: -90px;
+  }
+
+  @include media($medium-screen + $width-nav-sidebar) {
+    height: $site-top;
+    margin-top: -$site-top;
+  }
+}
+
 .usa-site-header {
   background-color: $color-white;
   border-bottom: 1px solid $color-gray-light;
+  position: fixed;
```

Comment:
I tested on iOS 8.4 (iPad & iPhone) and it seemed to work well on there. Do not have access to an Android device to test on there.